neondatabase/website
# partners page update

This pull request brings update to the partners page.

Steps to check:
- open [preview](https://neon-next-git-partners-page-upd-neondatabase.vercel.app/partners)
- make sure that everything looks and works as expected
